nucleus/core/kernel/src/main/java/org/glassfish/runnablejar/commandline/WordWrapper.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package org.glassfish.runnablejar.commandline;
+
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.function.BinaryOperator;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Collector;
+
+/**
+ *
+ * @author Ondro Mihalyi
+ */
+public class WordWrapper {
+
+    final int MAX_LINE_LENGTH;
+    final String HELP_LINE_INDENT;
+    int characterCount;
```

Comment:
I'd rather have it as local.

Reply:
This is an instance field, not a local variable. It's initialized based on constructor argument `initialIndent`.

Reply:
This must be an instance field, not a local variable. It holds the state. I refactored this class into a direct collector. That should make it clear it's a stateful class and shouldn't be reused.